lib/metrician/middleware/request_timing.rb

```diff
@@ -143,7 +143,7 @@ def self.extract_route(controller:, path:)
       end
 
       def self.get_response_size(headers:, body:)
-        return headers[HEADER_CONTENT_LENGTH] if headers[HEADER_CONTENT_LENGTH]
+        return headers[HEADER_CONTENT_LENGTH] if headers && headers[HEADER_CONTENT_LENGTH]
```

Comment:
Is there a way to test that without raising an exception in the test? Or is the only time it happens?